app.js

```diff
@@ -1,11 +1,23 @@
-import { fetchGithubRepos } from './GithubApi.js';
+import { fetchGithubIssues, fetchGithubRepos } from './GithubApi.js';
+import { formatDateTime } from './utils.js';
 
 let form = document.querySelector('form');
 let dialog = document.querySelector('.details-dialog');
-let dialogContent = dialog.querySelector('.details-dialog__content');
+let dialogContent = dialog.querySelector('.details-dialog__content__dynamic');
 let dinamicContent = document.querySelector('#dinamicContent');
 let skeletonScreen = document.querySelector('.skeleton-screen');
 
+dialog.addEventListener('click', (event) => {
+    if (event.target === dialog) {
+        dialog.close();
+        // The dialog takes the whole page and its content, only 40%.
```

Comment:
1. The comments need to go before the code they reference. In this case, between lines 10-11
2. A better comment would be:

> The native `<dialog>` element is filling up the whole page. So we can add an EventListener on the dialog, this will contain the entire page. But how do we figure out if we clicked inside the dialog content or on the "backdrop". To do this, we wrapped all the dialog elements inside a `<div>`. This way, when we click inside the dialog content, the `event.target` will reference this `<div>`. But if we click outside, it will reference the dialog itself. Thus, the `if` statement below is what we need to detect when to close the dialog.